src/main/java/com/nhantran/pages/BasePage.java

```diff
@@ -8,18 +9,19 @@ public class BasePage {
 
     private String railwayTab = "//div[@id='menu']//li/a/span[text()='%s']";
```

Comment:
rename selector variable to easier distigush, it's element (e/x: dynamicMenuTabSelector,...)